src/energy_corrections.F

```diff
@@ -2122,13 +2170,88 @@ SUBROUTINE ec_properties(qs_env, ec_env)
          END IF
 
          CALL cp_print_key_finished_output(unit_nr=unit_nr, logger=logger, &
-                                           basis_section=qs_env%input, print_key_path="DFT%PRINT%MOMENTS")
+                                           basis_section=ec_section, print_key_path="PRINT%MOMENTS")
          CALL get_qs_env(qs_env=qs_env, results=results)
          description = "[DIPOLE]"
          CALL cp_results_erase(results=results, description=description)
          CALL put_results(results=results, description=description, values=tdip(1:3))
       END IF
 
+      ! Do a Voronoi Integration or write a compressed BQB File
+      print_key_voro => section_vals_get_subs_vals(ec_section, "PRINT%VORONOI")
+      print_key_bqb => section_vals_get_subs_vals(ec_section, "PRINT%E_DENSITY_BQB")
+      IF (BTEST(cp_print_key_should_output(logger%iter_info, print_key_voro), cp_p_file)) THEN
+         should_print_voro = 1
+      ELSE
+         should_print_voro = 0
+      END IF
+      IF (BTEST(cp_print_key_should_output(logger%iter_info, print_key_bqb), cp_p_file)) THEN
+         should_print_bqb = 1
+      ELSE
+         should_print_bqb = 0
+      END IF
+      IF ((should_print_voro /= 0) .OR. (should_print_bqb /= 0)) THEN
+
+         CALL get_qs_env(qs_env=qs_env, &
+                         pw_env=pw_env)
+         CALL pw_env_get(pw_env=pw_env, &
+                         auxbas_pw_pool=auxbas_pw_pool, &
+                         pw_pools=pw_pools)
+         CALL pw_pool_create_pw(pool=auxbas_pw_pool, &
+                                pw=rho_elec_rspace%pw, &
+                                use_data=REALDATA3D, &
+                                in_space=REALSPACE)
+
+         IF (dft_control%nspins > 1) THEN
+
+            ! add Pout and Pz
+            CALL pw_copy(ec_env%rhoout_r(1)%pw, rho_elec_rspace%pw)
+            CALL pw_axpy(ec_env%rhoout_r(2)%pw, rho_elec_rspace%pw)
+
+            CALL pw_axpy(ec_env%rhoz_r(1)%pw, rho_elec_rspace%pw)
+            CALL pw_axpy(ec_env%rhoz_r(2)%pw, rho_elec_rspace%pw)
+
+            mb_rho => rho_elec_rspace%pw
+            !CALL voronoi_analysis( qs_env, rho_elec_rspace%pw, print_key, unit_nr )
```

Comment:
Why are these calls to `voronoi_analysis` commented out?

Reply:
The original code in qs_scf_post_gpw.F was refactured at some point. I kept the artifact around for no good reason.